src/realtime/flood/flood_hazard.py

```diff
@@ -149,23 +161,32 @@ def write_keywords(self):
         }
 
         keywords = {
-            'hazard_category': hazard_category_single_event['key'],
```

Comment:
cc @ismailsunni 
These following lines were refactored using safe.definitions.

For title, I thought we have fixed keys? But I didn't find it.